Sources/SwiftGodot/Core/RawCall.swift

```diff
@@ -0,0 +1,45 @@
+@_implementationOnly import GDExtension
+
+extension Object {
+
+    /// Make a raw call to a method bind.
+    /// All input arguments must be marshaled into `Variant`s.
+    /// The result is a `Variant` that must be unmarshaled into the expected type.
+    @discardableResult /* discardable per discardableList: Object, emit_signal */
+    final func rawCall(_ p_method_bind: GDExtensionMethodBindPtr, arguments: [Variant]) -> Variant? {
```

Comment:
I can't help feeling that there's a better way to do this!
Calls to the API are all generated, with the object handle as the first parameter, and the other arguments as a variadic parameter.

I needed to instead make a call that takes an array of arguments.